docs/obis-observer/types.md

```diff
@@ -0,0 +1,211 @@
+        <tr>
+            <td colspan="4"><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+        </tr>
+        <tr>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+        </tr>
+        <tr>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+        </tr>
+        <tr>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>1</code></td>
+        </tr>
+    </tbody>
+</table>
+
+Result: `1-0:11.35.0*0` in hex `0x08010b23`.
+
+
+Another example - for OBIS code `7-0:41.0.0*255`
+`5` bytes are necessary for this.
+
+The group values:
+
+| group | value | hex    | bits         |
+| ----- | ----- | ------ | ------------ |
+| `A`   | `7`   | `0x07` | `0b111`      |
+| `B`   | `0`   |        |              |
+| `C`   | `41`  | `0x29` | `0b101001`   |
+| `D`   | `0`   | `0x00` | `0b0`        |
+| `E`   | `0`   |        |              |
+| `F`   | `255` | `0xff` | `0b11111111` |
+
+Bytes representation in table form:
+
+<table>
+    <thead>
+        <tr>
+            <th>8</th>
+            <th>7</th>
+            <th>6</th>
+            <th>5</th>
+            <th>4</th>
+            <th>3</th>
+            <th>2</th>
+            <th>1</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td colspan="4"><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+        </tr>
+        <tr>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>1</code></td>
+            <td><code>1</code></td>
+        </tr>
+        <tr>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+            <td><code>0</code></td>
+            <td><code>0</code></td>
+            <td><code>1</code></td>
+        </tr>
+        <tr>
+            <td colspan="8" align="center"><code>0</code></td>
```

Comment:
maybe it's better not to join cells here?

Reply:
reworked as 8 cells of bits